addons/medical/functions/fnc_handleFallDamage.sqf

```diff
@@ -0,0 +1,14 @@
+// by commy2
+#include "script_component.hpp"
+
+params ["_unit", "_newDamage"];
+
+private ["_totalDamageL", "_totalDamageR"];
+
+_totalDamageL = (_unit getHitPointDamage "HitLeftLeg") + _newDamage;
+_totalDamageR = (_unit getHitPointDamage "HitRightLeg") + _newDamage;
+
+_unit setHitPointDamage ["HitLeftLeg", _totalDamageL];
+_unit setHitPointDamage ["HitRightLeg", _totalDamageR];
```

Comment:
HitLeftLeg and HitRightLeg don't do anything as of Arma v1.53. Arma currently does not distinguish between Left and Right limbs but packs them into one "metalimb", HitLegs in this case. For the left and right arm it is HitHands. This will likely change in a future version if the dev team keeps their current plans.